src/main/java/io/outfoxx/swiftpoet/CodeWriter.kt

```diff
@@ -406,10 +408,59 @@ internal class CodeWriter constructor(
     }
   }
 
+  override fun close() {
+    out.close()
+  }
+
   /**
    * Returns the modules that should have been imported for this code.
    */
-  fun suggestedImports(): Map<String, DeclaredTypeName> {
+  private fun suggestedImports(): Map<String, DeclaredTypeName> {
```

Comment:
Why is this private now?

Reply:
it doesnt need to be exposed? same inspiration as https://github.com/square/kotlinpoet/pull/1353